docs/API.md

```diff
@@ -1121,3 +1125,33 @@ A endpoint that can be used without the need for [authentication](#auth) which w
 HTTP GET /health
 
 It will output the state for each nodes in JSON, whose format might change in the future.
+
+## <a name="liquid"></a>Liquid integration
+
+NBXplorer supports liquid, the API is the same as all the other coins, except for the following:
+
+* All references to `value` which normally contains an integer of the amount of the altcoin will instead output a JSON Object of type `AssetMoney`.
+* [When listing the transaction of a derivation scheme](#transactions), the `balanceChange` elements is instead a `JSON array of AssetMoney`.
+* [Get Balance](#balance) returns values as `JSON array of AssetMoney`.
+* [Get a new unused address](#unused) returns a confidential address. (See note below)
+* [Create Partially Signed Bitcoin Transaction](#psbt) is not supported.
+* [Update Partially Signed Bitcoin Transaction](#updatepsbt) is not supported
+* [Scan UTXO Set](#scanUtxoSet) is not supported.
+* Any sort of recovery is not supported.
+
+The `AssetMoney` JSON format is:
+
+```json
+{
+	"assetId": "abc",
+	"value": 123
+}
+```
+
+The blinding key of the confidential address is derived directly from the `derivationScheme`.
+If the `scriptPubKey` `0/2` is generated, the blinding private key used by NBXplorer is the SHA256 of the scriptPubKey at `0/2/0`.
+
+In order to send in and out of liquid, we advise you to rely on the RPC command line interface of the liquid deamon.
+For doing this you need to [Generate a wallet](#wallet) with `importAddressToRPC` and `savePrivateKeys` set to `true`.
+
+Be careful to not expose your NBXplorer server on internet, your private keys can be [retrieved trivially](#getmetadata).
```

Comment:
This sounds like you are revealing the wallet privkeys to anyone who is able to connect to your nbxplorer instance, which I assume is not the case. You're talking about people getting the xpubkey and deriving the blinding privkeys, right? Maybe "the private blinding keys, used to reveal the asset and value types of confidential transactions, can be [retrieved trivially](#getmetadata)"?

Reply:
Yes, NBX is not meant to be exposed publicly.   This does indeed allow anyone with access to your NBX instance to retrieve your private keys, when setting  `savePrivateKeys` & `importAddressToRPC`.

The main use-case for this generate wallet endpoint is to be able to create a wallet through NBX( generate a seed), track it, and be able to import to the Bitcoin/Elements/Shitcoin Core wallet(with `importAddressToRPC`) and use it to view balances or spend( with `savePrivateKeys`).

In terms of Liquid, this is mandatory currently, to not complicate the NBX codebase in terms of additional derivation scheme parsing options( which  there is a branch I have from before but added quite a lot of code for this PR)